.github/ISSUE_TEMPLATE/module_request.yml

```diff
@@ -0,0 +1,37 @@
+name: Module Request
```

Comment:
```suggestion
name: Module Addition Request
```

Reply:
Should it be updated in the preset title as well?

Reply:
Sure. It would be nice to somehow reword it saying it is a module addition or listing request not module request (to be created)

Reply:
I think `Module Listing Request` is good